docs/configuration.md

```diff
@@ -886,6 +892,17 @@ and proxy connections to the registry server.
        Absolute path to x509 cert file
     </td>
   </tr>
+  <tr>
+    <td>
+      <code>rootcas</code>
+    </td>
+    <td>
+      yes
+    </td>
+    <td>
+      An array of paths to the root CA files, as x509.
```

Comment:
Great. So, could you clarify it? Please see Go's documentation for clarification:

```
// RootCAs defines the set of root certificate authorities
// that clients use when verifying server certificates.
// If RootCAs is nil, TLS uses the host's root CA set.
```

If the documentation is not clear, the feature will not be added.